packages/ui-core/theme/src/lib/components/theme-sidebar/theme-settings/components/theme-language-selector/theme-language-selector.component.scss

```
@@ -45,3 +45,14 @@
   align-items: center;
   flex-wrap: wrap;
 }
+
+// Outside nb-install-component on purpose: nb-option content renders in the
+// overlay (outside the host element), where :host-scoped rules can't reach.
```

Stylelint (17.14.0)

[error] Unclosed string (CssSyntaxError)

(CssSyntaxError)